src/Icon/Icon.js

```diff
@@ -0,0 +1,115 @@
+// @flow weak
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('Icon', (theme) => {
+  const { palette } = theme;
+  return {
+    accent: {
```

Comment:
Would having a `userSelect: 'none',` style by default make sen? I'm wondering if users should be able to select the text. I'm not sure.